acme/acme/messages.py

```diff
@@ -340,11 +340,45 @@ def from_data(cls, phone=None, email=None, external_account_binding=None, **kwar
 
         return cls(**kwargs)
 
+    def __init__(self, **kwargs):
```

Comment:
I think I can do that, but I don't quite follow what you mean. The code doesn't currently set contact, exactly, but sets a private variable about the presence of contact. I can certainly expand the existing brief comment on line 344 to explain the overall thinking, but I want to make sure I'm not misunderstanding where you'd like to see a comment....

Reply:
Sorry. GitHub won't let me comment on where I was referring to because you didn't modify the code there.

I was proposing we add a comment above [the definition of the contact field](https://github.com/certbot/certbot/blob/2a047eb526ee02d4bb3bb3668260ac6d481b99d3/acme/acme/messages.py#L318). My thinking is just to flag that the field no longer works as a normal field with the value of `omitempty=True` where the field is defined in case people miss your code here.